README.md

```diff
@@ -51,13 +51,17 @@ await element.when('mousemove')
 ```js
 // Imperative
 const controller = new AbortController();
-element.addEventListener('mousemove', e => {
-  console.log(e);
-
-  element.addEventListener('mouseup', e => {
-    controller.abort();
-  });
-}, { signal: controller.signal });
+element.addEventListener(
+	'mousemove',
+	(e) => {
+		console.log(e);
+
+		element.addEventListener('mouseup', (e) => {
+			controller.abort();
+		});
+	},
+	{ signal: controller.signal },
```

Comment:
Let's revert changes like this which are just stylistic and probably automated by an editor.

Reply:
They're from our committed prettier config.

Reply:
Can you revert them or only format the changes relevant to this PR instead of the whole file? That'd make it easier to review and better scoped.